stingray/utils.py

```diff
@@ -116,6 +116,88 @@ def rebin_data(x, y, dx_new, method='sum'):
     return xbin, ybin, step_size
```

Comment:
I think adding (and subtracting ) offset should be consistent with `rebin_data_lc` (mid bins )